cmd/templ/fmtcmd/testdata.txtar

```diff
@@ -32,3 +32,23 @@ templ b() {
 		</p>
 	</div>
 }
+-- c.templ --
```

Comment:
this example is a file that should not be changed by the formatter, to validate that it does not fail if files are formatted correctly.